libraries/AP_HAL_ChibiOS/hwdef/PixPilot-V6PRO/hwdef-bl.dat

```diff
@@ -0,0 +1,60 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1160
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+env OPTIMIZE -Os
+PB3 LED_ACTIVITY OUTPUT OPENDRAIN HIGH # red
+PE12 LED_BOOTLOADER OUTPUT OPENDRAIN HIGH # blue
+define HAL_LED_ON 0
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART2 USART3 UART7
+
+# telem1
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+
+# telem2
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+
+# UART7 maps to SERIAL5.
+PE7 UART7_RX UART7
+PE8 UART7_TX UART7
+
+
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# Add CS pins to ensure they are high in bootloader
+PC13 42688_EXT_CS CS
+PD7 BARO_EXT_CS CS
+PC2 40605_EXT_CS CS
+PE4 42688_CS CS
+PC14 BARO_CS CS 
+PD10 FRAM_CS CS
+#PD2 SDCARD_CS CS
+
+
+define HAL_USE_EMPTY_STORAGE 1
+define HAL_STORAGE_SIZE 32768
```

Comment:
need blank line at file end

also should add SD card to allow firmware load from SD

Reply:
OK，Have been modified.